teamcity/resource_build_config.go

```diff
@@ -128,6 +129,28 @@ func resourceBuildConfig() *schema.Resource {
 					},
 				},
 			},
+			"feature": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"properties": {
```

Comment:
Can this simply be changed to:

```
"properties": {
    Type:     schema.TypeMap,
    Required: true,
}
```

I also don't think this should be optional; if a feature is specified, it should always have properties specified as well, even that's an empty map.

Reply:
oops, I ended up preserving wrong version after squashing commits and merging.. luckily there are not many code changes